### ISSUE TITLE
Antivirus report

### ISSUE THREAD

**Issue by SeanHolt** [fiendish/aardwolfclientpackage#256]
AVG won't let the installer program run, no virus detected but blocks it as 'suspicious', partially because it is unsigned (certificates)... just a heads up

**Comment by fiendish** [comment]
As far as I know, signing the installers costs money.  :\

**Comment by SeanHolt** [comment]
Yes it does, I just wanted to let you know this occurred, just in case... You can also give a heads up on the site indicating why it occurs and that it is to be expected,

**Comment by fiendish** [comment]
That's a good point. I'll add something to the download page. Thanks.

**Comment by fiendish** [comment]
I'll leave this issue open in case someone either makes a credible free app signing authority or decides that they want to donate the hundreds of dollars to pay off the certificate mafia.

**Comment by fiendish** [comment]
I've added a link on https://fiendish.github.io/aardwolfclientpackage/ to a new Trust FAQ page
